vspreview/main/settings.py

```diff
@@ -131,6 +131,10 @@ def setup_ui(self) -> None:
             self.timeline_notches_margin_spinbox
         )
 
+        self.color_management_checkbox = QCheckBox(self)
+        self.color_management_checkbox.setText('Color management')
+        self.vlayout.addWidget(self.color_management_checkbox)
```

Comment:
Oh, I missed this: But if you only support color management on Win32, might as well hide the checkbox on other platforms.